prestodb/client.py

```diff
@@ -40,6 +40,7 @@
 import prestodb.logging
 import prestodb.redirect
 import requests
+from six.moves.urllib.parse import urlparse
```

Comment:
We want to remove dependency on six this can just be imported from urllib

Reply:
fixed under https://github.com/prestodb/presto-python-client/pull/101/commits/39e8b0dcf364c78c77e545847e891783ddba7ade